File: src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/EventOutcome.java
package uk.gov.justice.digital.hmpps.whereabouts.services;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
@JsonInclude(JsonInclude.Include.NON_NULL)
class EventOutcome {
    private final String eventOutcome;
    private final String performance;
}

File: src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/EventOutcome.kt
package uk.gov.justice.digital.hmpps.whereabouts.services

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class EventOutcome(
  val eventOutcome: String,
  val performance: String? = null,
  val outcomeComment: String? = null,
)
